src/main/resources/xslt/util/ddi/cleaning.xsl

```diff
@@ -123,7 +123,7 @@
                     <xsl:value-of select="concat($relative-path,$new-identifier)"/>
                     <!--                        <xsl:choose>
                             <!-\- for filters and controls in loops, fetching the nearest variable in the tree -\->
-                            <xsl:when test="ancestor::d:Loop | ancestor::d:QuestionGrid[d:GridDimension/d:Roster[not(@maximumAllowed)]]">
```

Comment:
Update on commented lines ?
A bit of cleaning in the commented code lines should be done.